lib/LedgerSMB/Admin/Command/user.pm

```diff
@@ -475,11 +481,11 @@ User name of the created or changed user
 
 Password
 
-=item start_date
+=item start-date
```

Comment:
Takes a date as an argument. But which format?

Reply:
Fixed in documentation